src/test/ui/imports/tool-mod-child.rs

```diff
@@ -0,0 +1,7 @@
+use clippy::a; //~ ERROR
```

Comment:
```suggestion
use clippy::a; //~ ERROR unresolved import `clippy`
```
Please use messages after `//~ ERROR` annotations, they are important to make tests resistant to mistakes.